htdocs/core/lib/functions.lib.php

```diff
@@ -1950,7 +1950,7 @@ function dol_get_fiche_head($links = array(), $active = '', $title = '', $notab
 			$tabsname = str_replace("@", "", $picto);
 		}
 		$out .= '<div id="moretabs'.$tabsname.'" class="inline-block tabsElem valignmiddle">';
-		$out .= '<div class="tab"><a href="#" class="tab moretab inline-block tabunactive"><span class="hideonsmartphone">'.$langs->trans("More").'</span>... ('.$nbintab.')</a></div>'; // Do not use "reposition" class in the "More".
+		$out .= '<div class="tab tabunactive"><a href="#" class="tab moretab inline-block"><span class="hideonsmartphone">'.$langs->trans("More").'</span><span class="badge marginleftonlyshort">'.$nbintab.'</span></a></div>'; // Do not use "reposition" class in the "More".
```

Comment:
The css "badge" is currently used to highlight a quantity of a data in database (related to text before). Here we want a number of menu entries. So i suggest to introduce another css
For example  "badgemenu" instead of "badge".

Reply:
I quite agree with the idea of using two different styles of badge because it does not correspond to the same thing.
However after analyzing other examples of tabs with a grouping, I see that they do not use a totalization and I wonder if we should not simply do without this function